core/common/src/main/java/alluxio/grpc/GrpcChannelBuilder.java

```diff
@@ -48,11 +48,14 @@
   protected AlluxioConfiguration mConfiguration;
 
   private GrpcChannelBuilder(SocketAddress address, AlluxioConfiguration conf) {
+    mConfiguration = conf;
     mChannelKey = GrpcManagedChannelPool.ChannelKey.create(conf);
+    // Set default overrides for the channel.
     mChannelKey.setAddress(address).usePlaintext();
+    mChannelKey.setMaxInboundMessageSize(
+        (int) mConfiguration.getBytes(PropertyKey.USER_NETWORK_MAX_INBOUND_MESSAGE_SIZE));
```

Comment:
Does this affect both the client and servers?

Reply:
Whoever acts as a client, and create a channel, will use this value. So both client connections and inter-connections will be affected.